data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/util/ContextMatcher.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression.util;
```

Comment:
FYI, I think we are trying to avoid `util` packages. Personally, I don't feel too strongly on this if they are isolated to testing code.